blackjax/kernels.py

```diff
@@ -106,9 +108,10 @@ def __new__(  # type: ignore[misc]
         mcmc_parameters: Dict,
         resampling_fn: Callable,
         mcmc_iter: int = 10,
+        with_parameter_tunning: Callable = no_tunning,
```

Comment:
I am not sure I like this design. Look at the separation between sampling and adaptation for the MCMC algorithm.